Server/src/main/java/com/bbd/BeanServer/controller/CommentController.java

```diff
@@ -0,0 +1,41 @@
+package com.bbd.BeanServer.controller;
+
+
+import com.bbd.BeanServer.service.CommentReactionService;
+import com.bbd.BeanServer.service.ReactionService;
+import com.bbd.shared.models.Reaction;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.crossstore.ChangeSetPersister;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import com.bbd.shared.models.CommentReaction;
+
+import java.util.Map;
+
+@RestController
+public class CommentController {
+    @Autowired
+    private CommentReactionService commentReactionService;
+
+    @Autowired
+    private ReactionService reactionService;
+
+
+    @PostMapping("/commentreaction")
+    public ResponseEntity<Void> commentReaction(@RequestBody Map<String, Object> requestBody) throws ChangeSetPersister.NotFoundException {
```

Comment:
Yea there is a reason, I have to pass in 2 objects being, Reaction and CommentReaction, didnt want to go with creating a wrapper object for these two, so went with a map which I have too deserialize, but if you think theres a cleaner way let me know.

Reply:
I see, I missed that.

I suppose I would have just made a wrapper, Java is very class-y with everything it does. This is personal preference though, so that's up to you.